v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -689,6 +696,73 @@ func (f *Frontend) processMessage(message string) {
 	}()
 }
 
+func (f *Frontend) processMessageWithAdditionalObjects(message string, sender *edge.ICoreWebView2, args *edge.ICoreWebView2WebMessageReceivedEventArgs) {
+	if strings.HasPrefix(message, "file:") {
+		callbackID := message[5:]
+
+		objs, err := args.GetAdditionalObjects()
+		defer objs.Release()
+
+		if err != nil {
+			f.logger.Error(err.Error())
+			return
+		}
+
+		if objs == nil {
+			return
+		}
+
+		count, err := objs.GetCount()
+		if err != nil {
+			f.logger.Error(err.Error())
+			return
+		}
+
+		files := []File{}
+		for i := uint32(0); i < count; i++ {
+			_file, err := objs.GetValueAtIndex(i)
+			defer func() {
+				err := _file.CallRelease(unsafe.Pointer(_file))
```

Comment:
If `GetValueAtIndex` has returned an error `_file` would be nil and this defer will then panic. I think it would be better to move it after the `if err != nil` block.